microservices/user/src/services.py

```diff
@@ -0,0 +1,131 @@
+from datetime import datetime
+from datetime import timedelta
+from typing import Dict
+
+from src.common.constants import ExceptionsMessages
+from src.common.constants import JWT_REFRESH_DELTA
+from src.common.decorators import db_session
+from src.common.exceptions import ResourceNotFoundException
+from src.common.logger import logger
+from src.models.company import Company
+from src.models.user import User
+from src.models.entities import CompanyResponseEntity
+from src.serializers import CompanyResponseSerializer, CompanyInputSerializer
+
+
+def insert_company(company_data):
+    company_input = CompanyInputSerializer().load_with_exception(company_data)
+    return insert_company_model(company_input)
+
+
+def get_company_by_id(idCompany: int):
```

Comment:
@Richardace please, change var names to match snake case.
See: [best practices](https://www.digitalogy.co/blog/top-python-best-practices-for-better-code-quality/)